core/src/main/java/edu/wpi/grip/core/exception/GripException.java

```diff
@@ -0,0 +1,26 @@
+
+package edu.wpi.grip.core.exception;
+
+/**
+ * An exception thrown when something goes wrong with an internal GRIP
+ * operation.
+ */
+public class GripException extends RuntimeException {
+
+    public GripException() {
```

Comment:
It might be a good idea to use this as as base class for internal exceptions and use it like `RuntimeException` - make subclasses for specific uses but never use it directly. So maybe have something like `GripServerException extends GripException`